File: MED-CLOUD/med-data/src/main/java/nta/med/data/model/ihis/cpls/CPL3010U00GrdPartListItemInfo.java
package nta.med.data.model.ihis.cpls;

import java.math.BigInteger;
import java.sql.Timestamp;

public class CPL3010U00GrdPartListItemInfo {
	private String labNo;
    private String specimenSer;
    private String bunho;
    private String suname2;
    private String fnBasLoadGwaName;
    private String hoDong;
    private Timestamp partJubsuDate;
    private String partJubsuTime;
    private String partJubsuja;
    private String jubsuja;
    private String remark;
    private String partJubsuFlag;
    private BigInteger fkocs;
    private String inOutGubun;
    private String doctorName;
    private String tubeName;
    private String emergency;
    private String sunabYn;
    private String resultYn;
    private String labelOneMore;
	public CPL3010U00GrdPartListItemInfo(String labNo, String specimenSer,
			String bunho, String suname2, String fnBasLoadGwaName,
			String hoDong, Timestamp partJubsuDate, String partJubsuTime,
			String partJubsuja, String jubsuja, String remark,
			String partJubsuFlag, BigInteger fkocs, String inOutGubun,
			String doctorName, String tubeName, String emergency,
			String sunabYn, String resultYn, String labelOneMore) {
		super();
		this.labNo = labNo;
		this.specimenSer = specimenSer;
		this.bunho = bunho;
		this.suname2 = suname2;
		this.fnBasLoadGwaName = fnBasLoadGwaName;
		this.hoDong = hoDong;
		this.partJubsuDate = partJubsuDate;
		this.partJubsuTime = partJubsuTime;
		this.partJubsuja = partJubsuja;
		this.jubsuja = jubsuja;
		this.remark = remark;
		this.partJubsuFlag = partJubsuFlag;
		this.fkocs = fkocs;
		this.inOutGubun = inOutGubun;
		this.doctorName = doctorName;
		this.tubeName = tubeName;
		this.emergency = emergency;
		this.sunabYn = sunabYn;
		this.resultYn = resultYn;
		this.labelOneMore = labelOneMore;
	}
	public String getLabNo() {
		return labNo;
	}
	public void setLabNo(String labNo) {
		this.labNo = labNo;
	}
	public String getSpecimenSer() {
		return specimenSer;
	}
	public void setSpecimenSer(String specimenSer) {
		this.specimenSer = specimenSer;
	}
	public String getBunho() {
		return bunho;
	}
	public void setBunho(String bunho) {
		this.bunho = bunho;
	}
	public String getSuname2() {
		return suname2;
	}
	public void setSuname2(String suname2) {
		this.suname2 = suname2;
	}
	public String getFnBasLoadGwaName() {
		return fnBasLoadGwaName;
	}
	public void setFnBasLoadGwaName(String fnBasLoadGwaName) {
		this.fnBasLoadGwaName = fnBasLoadGwaName;
	}
	public String getHoDong() {
		return hoDong;
	}
	public void setHoDong(String hoDong) {
		this.hoDong = hoDong;
	}
	public Timestamp getPartJubsuDate() {
		return partJubsuDate;
	}
	public void setPartJubsuDate(Timestamp partJubsuDate) {
		this.partJubsuDate = partJubsuDate;
	}
	public String getPartJubsuTime() {
		return partJubsuTime;
	}
	public void setPartJubsuTime(String partJubsuTime) {
		this.partJubsuTime = partJubsuTime;
	}
	public String getPartJubsuja() {
		return partJubsuja;
	}
	public void setPartJubsuja(String partJubsuja) {
		this.partJubsuja = partJubsuja;
	}
	public String getJubsuja() {
		return jubsuja;
	}
	public void setJubsuja(String jubsuja) {
		this.jubsuja = jubsuja;
	}
	public String getRemark() {
		return remark;
	}
	public void setRemark(String remark) {
		this.remark = remark;
	}
	public String getPartJubsuFlag() {
		return partJubsuFlag;
	}
	public void setPartJubsuFlag(String partJubsuFlag) {
		this.partJubsuFlag = partJubsuFlag;
	}
	public BigInteger getFkocs() {
		return fkocs;
	}
	public void setFkocs(BigInteger fkocs) {
		this.fkocs = fkocs;
	}
	public String getInOutGubun() {
		return inOutGubun;
	}
	public void setInOutGubun(String inOutGubun) {
		this.inOutGubun = inOutGubun;
	}
	public String getDoctorName() {
		return doctorName;
	}
	public void setDoctorName(String doctorName) {
		this.doctorName = doctorName;
	}
	public String getTubeName() {
		return tubeName;
	}
	public void setTubeName(String tubeName) {
		this.tubeName = tubeName;
	}
	public String getEmergency() {
		return emergency;
	}
	public void setEmergency(String emergency) {
		this.emergency = emergency;
	}
	public String getSunabYn() {
		return sunabYn;
	}
	public void setSunabYn(String sunabYn) {
		this.sunabYn = sunabYn;
	}
	public String getResultYn() {
		return resultYn;
	}
	public void setResultYn(String resultYn) {
		this.resultYn = resultYn;
	}
	public String getLabelOneMore() {
		return labelOneMore;
	}
	public void setLabelOneMore(String labelOneMore) {
		this.labelOneMore = labelOneMore;
	}
}

File: IHIS/IHIS.Framework/IHIS.CloudConnector/Contracts/Models/Cpls/CPL3010U00GrdPartListItemInfo.cs
using System;

namespace IHIS.CloudConnector.Contracts.Models.Cpls
{
    [Serializable]
	public class CPL3010U00GrdPartListItemInfo
	{
		private String _labNo;
		private String _specimenSer;
		private String _bunho;
		private String _suname2;
		private String _fnBasLoadGwaName;
		private String _hoDong;
		private String _partJubsuDate;
		private String _partJubsuTime;
		private String _partJubsuja;
		private String _jubsuja;
		private String _remark;
		private String _partJubsuFlag;
		private String _fkocs;
		private String _inOutGubun;
		private String _doctorName;
		private String _tubeName;
		private String _emergency;
		private String _sunabYn;
		private String _resultYn;
		private String _labelOneMore;
		private String _dataRowState;

		public String LabNo
		{
			get { return this._labNo; }
			set { this._labNo = value; }
		}

		public String SpecimenSer
		{
			get { return this._specimenSer; }
			set { this._specimenSer = value; }
		}

		public String Bunho
		{
			get { return this._bunho; }
			set { this._bunho = value; }
		}

		public String Suname2
		{
			get { return this._suname2; }
			set { this._suname2 = value; }
		}

		public String FnBasLoadGwaName
		{
			get { return this._fnBasLoadGwaName; }
			set { this._fnBasLoadGwaName = value; }
		}

		public String HoDong
		{
			get { return this._hoDong; }
			set { this._hoDong = value; }
		}

		public String PartJubsuDate
		{
			get { return this._partJubsuDate; }
			set { this._partJubsuDate = value; }
		}

		public String PartJubsuTime
		{
			get { return this._partJubsuTime; }
			set { this._partJubsuTime = value; }
		}

		public String PartJubsuja
		{
			get { return this._partJubsuja; }
			set { this._partJubsuja = value; }
		}

		public String Jubsuja
		{
			get { return this._jubsuja; }
			set { this._jubsuja = value; }
		}

		public String Remark
		{
			get { return this._remark; }
			set { this._remark = value; }
		}

		public String PartJubsuFlag
		{
			get { return this._partJubsuFlag; }
			set { this._partJubsuFlag = value; }
		}

		public String Fkocs
		{
			get { return this._fkocs; }
			set { this._fkocs = value; }
		}

		public String InOutGubun
		{
			get { return this._inOutGubun; }
			set { this._inOutGubun = value; }
		}

		public String DoctorName
		{
			get { return this._doctorName; }
			set { this._doctorName = value; }
		}

		public String TubeName
		{
			get { return this._tubeName; }
			set { this._tubeName = value; }
		}

		public String Emergency
		{
			get { return this._emergency; }
			set { this._emergency = value; }
		}

		public String SunabYn
		{
			get { return this._sunabYn; }
			set { this._sunabYn = value; }
		}

		public String ResultYn
		{
			get { return this._resultYn; }
			set { this._resultYn = value; }
		}

		public String LabelOneMore
		{
			get { return this._labelOneMore; }
			set { this._labelOneMore = value; }
		}

		public String DataRowState
		{
			get { return this._dataRowState; }
			set { this._dataRowState = value; }
		}

		public CPL3010U00GrdPartListItemInfo() { }

		public CPL3010U00GrdPartListItemInfo(String labNo, String specimenSer, String bunho, String suname2, String fnBasLoadGwaName, String hoDong, String partJubsuDate, String partJubsuTime, String partJubsuja, String jubsuja, String remark, String partJubsuFlag, String fkocs, String inOutGubun, String doctorName, String tubeName, String emergency, String sunabYn, String resultYn, String labelOneMore, String dataRowState)
		{
			this._labNo = labNo;
			this._specimenSer = specimenSer;
			this._bunho = bunho;
			this._suname2 = suname2;
			this._fnBasLoadGwaName = fnBasLoadGwaName;
			this._hoDong = hoDong;
			this._partJubsuDate = partJubsuDate;
			this._partJubsuTime = partJubsuTime;
			this._partJubsuja = partJubsuja;
			this._jubsuja = jubsuja;
			this._remark = remark;
			this._partJubsuFlag = partJubsuFlag;
			this._fkocs = fkocs;
			this._inOutGubun = inOutGubun;
			this._doctorName = doctorName;
			this._tubeName = tubeName;
			this._emergency = emergency;
			this._sunabYn = sunabYn;
			this._resultYn = resultYn;
			this._labelOneMore = labelOneMore;
			this._dataRowState = dataRowState;
		}

	}
}